jupyterbook/_toc.yml

```diff
@@ -1,68 +1,64 @@
-- file: content/intro
+- file: content/intro.md
   title: The IOOS Data Demo Center
 
 - part: Code Gallery
   chapters:
 
-    - file: content/Code Gallery/data_access/data_access.md
+    - file: content/Code Gallery/data_access.md
       sections:
-          - file: content/Code Gallery/data_access/2016-10-12-fetching_data.ipynb
-          - file: content/Code Gallery/data_access/2017-05-14-running_compliance_checker.ipynb
-          - file: content/Code Gallery/data_access/2016-12-20-searching_glider_deployments.ipynb
-          - file: content/Code Gallery/data_access/2017-01-18-siphon-explore-thredds.ipynb
-          - file: content/Code Gallery/data_access/2017-03-21-ERDDAP_IOOS_Sensor_Map.ipynb
-          - file: content/Code Gallery/data_access/2017-06-12-NCEI_RA_archive_history.ipynb
-          - file: content/Code Gallery/data_access/2017-07-25-HFRadar_currents.ipynb
-          - file: content/Code Gallery/data_access/2017-12-15-finding_HFRadar_currents.ipynb
-          - file: content/Code Gallery/data_access/2018-03-01-erddapy.ipynb
+          - file: content/Code Gallery/data_access_notebooks/2016-10-12-fetching_data.ipynb
```

Comment:
If we are going to use the directory to create the TOC file automatically we need to keep them, right?
Unless you have some a different idea.

Reply:
Yes. I think we do. Will have to check though while writing the script.